src/rosdistro/distribution.py

```diff
@@ -32,22 +32,30 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 from .manifest_provider.bitbucket import bitbucket_manifest_provider
-from .manifest_provider.git import git_manifest_provider
+from .manifest_provider.git import git_manifest_provider, git_source_manifest_provider
 from .manifest_provider.github import github_manifest_provider
```

Comment:
There's probably an argument to be made here for moving these to classes with static methods (`GitProvider` with `get_release_package_xml` and `get_source_package_xmls`). I wanted to avoid getting into a big refactoring here, but can do so if there's consensus that that's the way to go.